admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,52 @@
 How to Upgrade Your ownCloud Server
 ===================================
 
-There are three ways to upgrade your ownCloud server:
+We recommend that you keep your ownCloud server upgraded regularly installing all point releases and major releases, without skipping any of them, as doing so increases the risk of errors. 
+
+- Major releases are 8.0, 8.1, 8.2, 9.0, 9.1, and 10.0. 
+- Point releases are intermediate releases for each major release, e.g., 8.0.9, 8.1.3, and 10.0.2. 
+
+When an update is available for your ownCloud server, you will see a notification at the top of your ownCloud Web interface. 
```

Comment:
AFAIK there are still issues with the update notifications as long as the updates via the updater app are delayed. Only once the update/upgrade via the update server is provided there will be a notification.